desktop/cmp/treemap/TreeMap.js

```diff
@@ -103,24 +104,23 @@ class LocalModel extends HoistModel {
         this.clickCount = 0;
         this.debouncedClickHandler = debounce(this.clickHandler, 500);
 
-        // Render HighChart when chartElem container ready in DOM, a relevant observable on the
-        // model API changes, and/or the source data array itself changes.
+        // Render HighChart when chartElem container ready in DOM, or dependencies updated
+        const chartDependencies = () => ([
+            this.chartRef.current,
+            model.highChartsConfig,
+            model.data,
+            XH.darkTheme
+        ]);
```

Comment:
We're missing the dependency on the `model.algorithm` here. The chart needs to be redrawn when the layout algorithm changes

Reply:
restored